Atomic/syscontainers.py

```diff
@@ -296,9 +296,9 @@ def _get_remote_location(self, remote_input):
         remote_path = self._resolve_remote_path(remote_input)
         if remote_path:
             remote_rootfs = os.path.sep.join([remote_path, "rootfs"])
-            if os.path.exists(remote_path):
+            if os.path.exists(remote_rootfs):
```

Comment:
This is changed because when remote is `xx/rootfs` and `/xx/rootfs/usr` exist, because remote itself exist, it does not mean `xx/rootfs/rootfs`(remote rootfs) exist. 

As a result before case will fail and return a wrong remote path location: i.e (/xx/rootfs).(which I think is wrong).  It is because before the `if os.path.exists(remote_path)` will be true as `/xx/rootfs`(remote_path) exist